blog/_posts/2020-09-09-github-issues.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Issues? GitHub!"
+layout: archive
+author: Axel Naumann
+---
+
+First of all: a big THANK YOU for submitting bugs for ROOT.
+We appreciate the effort: finding back the issue submission web page, explaining what the problem is, and sometimes even providing a reproducer.
+
+To make it easier for you to submit issues, we now switched from ~~Jira~~ to [GitHub Issues](https://github.com/root-project/root/issues/new).
+We hope that having a GitHub account is so common these days that you're already logged in and used to GitHub's interfaces.
+
+O, and Markdown instead of `{code}`. Phew!
```

Comment:
```suggestion
Oh, and Markdown instead of `{code}`. Phew!
```

Reply:
Many people won't understand what this is about. They are not in our bubble.

Reply:
https://en.wikipedia.org/wiki/O_Captain!_My_Captain! ;-)

That said, "oh" is better and striking this sentence is best! Thanks!

Reply:
it's two different O[h]'s! :D ttps://www.mentalfloss.com/article/56582/whats-difference-between-o-and-oh